File: Cs-Connect-java/app/src/main/java/com/csestateconnect/www/csconnect/FilterAdapterCallback.java
package com.csestateconnect.www.csconnect;

public interface FilterAdapterCallback {
    void onBhkcheckedCallback(Integer position, Boolean checkedValue);
    void onStatuscheckedCallback(Integer position, Boolean checkedValue);
    void onAmenitycheckedCallback(Integer position, Boolean checkedValue);
    void onLocationcheckedCallback(Integer position, Boolean checkedValue);
}

File: CsConnect/app/src/main/java/com/csestateconnect/adapters/projectAdapters/project_filter_adapters/FilterAdapterCallback.kt
package com.csestateconnect.adapters.projectAdapters.project_filter_adapters

interface FilterAdapterCallback {

    fun onUnitCheckedCallback(position: Int?, checkedValue: Boolean?)
    fun onStatusCheckedCallback(position: Int?, checkedValue: Boolean?)
    fun onAmenityCheckedCallback(position: Int?, checkedValue: Boolean?)
    fun onLocationCheckedCallback(position: Int, checkedValue: Boolean)
}
